### ISSUE TITLE
Losses sharply decrease during training.

### ISSUE THREAD

**Issue by fitzjalen** [MIC-DKFZ/nnUNet#2835]
I'm training gallstone and kidney stone model with pretrained weights and QuaterLR (using 250 epochs) and got this sharp loss decrease. It's not first time I see this. noSmooth trainer works - but has worse DICE.

![Image](https://github.com/user-attachments/assets/85f39a8b-5c88-4918-a924-2069855c17bd)

![Image](https://github.com/user-attachments/assets/5d04babc-099f-452f-864b-056a4d025fd3)

**Comment by fitzjalen** [comment]
Oh yes, 3d_fullres, CT images, 375 cases, patch_size 192, bs 2

**Comment by constantinulrich** [comment]
Hi, sorry for the late reply. To be honest, we also do not really know why this sometimes happens. It mostly happens for small target structures and cases with no foreground. One explanation could be that the network learns not to predict anything for cases with no foreground.
